agent/engine/docker_container_engine.go

```diff
@@ -97,7 +97,7 @@ type DockerClient interface {
 	Version() (string, error)
 	InspectImage(string) (*docker.Image, error)
 	RemoveImage(string, time.Duration) error
-	LoadImage(docker.LoadImageOptions, time.Duration) error
+	LoadImage(io.Reader, time.Duration) error
```

Comment:
Can you also return the `Image` object here returned by the docker inspect in `validatePauseContainerImage`? So that I can to use the image name and ID to whitelist the pause image in image cleanup for free.